docs/swagger.json

```diff
@@ -965,43 +984,41 @@
         "responses": {}
       }
     },
-    "/pinning/pins/{id}": {
-      "post": {
-        "description": "This endpoint replaces a pinned object.",
+    "/pinning/pins/{requestid}": {
```

Comment:
Let's standardize this as `pinid`

Reply:
Isn't it better to use `requestid`? Because this `id` is named as `requestid` in `IpfsPinStatus` struct and in also [corresponding IPFS docs](https://ipfs.github.io/pinning-services-api-spec/#tag/pins/paths/~1pins~1{requestid}/get).

Reply:
I prefer `pinid` honestly, I feel like it tells more of a story. But I'm not super attached to either of those. @en0ma what do you think?